src/models/AuthenticationUtils.ts

```diff
@@ -1,10 +1,10 @@
 import { isNullOrUndefined } from "util";
-import * as core from '@actions/core';
 import { execFile } from "child_process";
 import { FetchCallType, ContentTypeMap, TokenScope } from "./UtilModels";
 import { jwtDecode, JwtPayload } from "jwt-decode";
 import { IHeaders } from "typed-rest-client/Interfaces";
 import * as InputConstants from "./InputConstants";
+import * as CoreUtils from '@actions/core';
```

Comment:
This import seems wrong? It should be `import CoreUtils from "./CoreUtils";` ?

Reply:
I see this is not even using the utils you wrote

Reply:
+1

Reply:
done.